docs/build

```diff
@@ -0,0 +1 @@
+../docs/docsite/_build
```

Comment:
this is weird. why is this not `./docsite/_build` then? also, why can't this be fixed on Zuul side? I'd rather not have symlinks in this repo

Reply:
I can change the location using a parameter for jobs defined inside the repo. For the moment I am only trying to see if new location is enough. 

If I can avoid the symlink, I will do it.